### ISSUE TITLE
question about darknet19 accuracy

### ISSUE THREAD

**Issue by kakpple** [wenxichen/tensorflow_yolo2#4]
hi, 
So you said the accuracy was around ~60%, you mean top-1 accuracy on imagenet validation set?
Accuracy should be above 70% as claimed in paper.

I am now training darknet-19 basenet using tensorflow slim. and get accuracy about 60%.

Any idea on this isseue?
